addons/website_livechat/__manifest__.py

```diff
@@ -37,7 +37,6 @@
         ],
         'web.assets_frontend': [
             'website_livechat/static/src/legacy/public_livechat.js',
-            'website_livechat/static/src/legacy/website_livechat_chatbot_test_script.js',
```

Comment:
Just forgot to remove in the manifest.
Made runbot red, but for some reasons had no issue locally even with `-u`